### PR TITLE
kmymoney, new recipe

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

**Comment by Begasus** [comment]
Needs https://github.com/haikuports/haikuports/pull/9796 to be merged first.

File: app-office/kmymoney/kmymoney-5.1.3.recipe [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="!x86"
+
+libVersion="$portVersion"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	kmymoney$secondaryArchSuffix = $portVersion
+	cmd:kmymoney$secondaryArchSuffix = $portVersion
```
Comment (jmairboeck) [inline app-office/kmymoney/kmymoney-5.1.3.recipe:22]: Shouldn't this use the `app:` prefix?

```suggestion
	app:KMyMoney = $portVersion
```

Comment (Begasus) [reply app-office/kmymoney/kmymoney-5.1.3.recipe:22]: Thanks, will update in the morning :)

File: app-office/kmymoney/kmymoney-5.1.3.recipe [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="!x86"
+
+libVersion="$portVersion"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	kmymoney$secondaryArchSuffix = $portVersion
+	app:KMyMoney$secondaryArchSuffix = $portVersion
```
Comment (jmairboeck) [inline app-office/kmymoney/kmymoney-5.1.3.recipe:22]: and once again: `$appsDir` isn't architecture-specific, IIRC, so:
```suggestion
	app:KMyMoney = $portVersion
```
(sorry for being pedantic ...)

Comment (Begasus) [reply app-office/kmymoney/kmymoney-5.1.3.recipe:22]: No problem, will fix :)

**Pull request merged**